.travis.yml

```diff
@@ -1,5 +1,17 @@
+language: python
+python:
+  - "3.8"
 services:
   - docker
-
+  - postgresql
+env:
```

Comment:
It would probably be a good idea to add `FLASK_ENV=testing` here. Docs on what that does are [here](https://flask.palletsprojects.com/en/1.1.x/config/#TESTING).

Reply:
I'll add that.

Reply:
Apparently according to the [travis documentation](https://docs.travis-ci.com/user/environment-variables#defining-public-variables-in-travisyml) "One build will be triggered for each line in the env array." so it treats them separately and the build fails without the DATABASE_URL set. So I'm not sure what to do here yet.

Reply:
Figured it out. They need to be on the same line.